kitsune/users/models.py

```diff
@@ -430,35 +431,31 @@ class AccountEvent(models.Model):
     # Status of an event entry.
     UNPROCESSED = 1
     PROCESSED = 2
-    ERRORED = 3
+    IGNORED = 3
+    NOT_IMPLEMENTED = 4
     EVENT_STATUS = (
         (UNPROCESSED, 'unprocessed'),
         (PROCESSED, 'processed'),
-        (ERRORED, 'errored'),
+        (IGNORED, 'ignored'),
+        (NOT_IMPLEMENTED, 'not-implemented'),
     )
 
-    PASSWORD_CHANGE = 1
-    PROFILE_CHANGE = 2
-    SUBSCRIPTION_STATE_CHANGE = 3
-    DELETE_USER = 4
-    EVENT_TYPE = (
-        (PASSWORD_CHANGE, 'password-change'),
-        (PROFILE_CHANGE, 'profile-change'),
-        (SUBSCRIPTION_STATE_CHANGE, 'subscription-state-change'),
-        (DELETE_USER, 'delete-user')
-    )
+    PASSWORD_CHANGE = 'password-change'
+    PROFILE_CHANGE = 'profile-change'
+    SUBSCRIPTION_STATE_CHANGE = 'subscription-state-change'
+    DELETE_USER = 'delete-user'
 
     status = models.PositiveSmallIntegerField(choices=EVENT_STATUS,
                                               default=UNPROCESSED,
                                               blank=True)
     created_at = models.DateTimeField(auto_now_add=True)
     last_modified = models.DateTimeField(auto_now=True)
-    events = models.TextField(max_length=4096, blank=False)
-    event_type = models.PositiveSmallIntegerField(choices=EVENT_TYPE,
-                                                  default=None,
-                                                  null=True,
-                                                  blank=False)
-    fxa_uid = models.CharField(blank=True, null=True, unique=True, max_length=128)
+    body = models.TextField(max_length=4096, blank=False)
+    event_type = models.CharField(max_length=256,
```

Comment:
:+1: for the change to charfield. It may be useful though to keep the event_type enum